entries/Wonderworm-Willy/game.json

```diff
@@ -1,6 +1,6 @@
 {
     "description": "Willy Wonderworm is a game for the Nintendo Gameboy and Gameboy Color systems. It should operate under most emulators and on an actual Gameboy system if copied onto a cartridge. Note that the scrolling text on the title screen may not appear correctly on some emulators and we recommend that you use the nocash emulator (no$gmb).",
```

Comment:
This entry does not have a working rom...

Reply:
It's not my fault, I didn't add it, it doesn't come from me.

However, I notice that it is already on HH under another page, and it perfectly works.
(here: https://hh.gbdev.io/game/willy-wonderworm)

Not sure how to proceed here:
1. remove the Wonderworm-Willy folder, add missing infos to willy-wonderworm (the correct one)
2. don't care about Wonderworm-Willy, add missing infos to willy-wonderworm (the correct one). We will later open a MR that will take care of removing Wonderworm-Willy.

Lemme know.

Reply:
No one has "faults" here.. but if the PR/Issue is about adding the entries from an event, for which a working link was provided, listing every entry with the corresponding ROMs, so a PR adding entries from such event should at least require you to go to every entry and make sure they are in the database. This didn't happen, as you added a tag to a broken entry ..

If I have to go through every diff you introduced to make sure you are applying a tag to the correct entry because you didn't even check what you are applying a tag to...well... it's better for me to restart the PR on my own

Reply:
Done!